core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -881,6 +884,16 @@ public void withStartupAttempts(int attempts) {
         this.startupAttempts = attempts;
     }
 
+    @Override
+    public void close() {
+        stop();
+    }
+
+    public SELF withCustomizer(Consumer<CreateContainerCmd> hook) {
```

Comment:
Funny thing - I initially named it `withCreateContainerCmdHook` and then was playing with different names and accidentally committed this one :D